extensions/amp-story-subscription/0.1/amp-story-subscription.js

```diff
@@ -0,0 +1,35 @@
+import {CSS} from '../../../build/amp-story-subscription-0.1.css';
+import {Layout_Enum} from '#core/dom/layout';
+import {htmlFor} from '#core/dom/static-template';
+
+const TAG = 'amp-story-subscription';
+
+// Create a paywall dialog element that have required attributes to be able to be
+// rendered by amp-subscriptions.
+// TODO: complete the rest of paywall dialog UI based on the publisher-provided attributes.
+const getPaywallDialogTemplate = (element) => {
+  return htmlFor(element)`
```

Comment:
Are we able to use JSX now?

Reply:
Something like this https://github.com/ampproject/amphtml/pull/36706

Reply:
Done.